client/src/app.jsx

```diff
@@ -7,10 +7,22 @@ import QandA from './components/questions/QandA.jsx'
 
 export const App = () => {
 
+  const [mainProduct, setMainProduct] = React.useState("37314");
+  const [mainProductName, setMainProductName] = React.useState( "Slacker's Slacks" )
+
+
+  const handleChangeProduct = (info) => {
+    console.log(info.id.toString(), info.name)
+
+      setMainProduct( info.id.toString() );
+      setMainProductName ( info.name);
```

Comment:
try to keep the spacing consistent ( either spaces on both sides of info.name or on neither side)